lib/api.js

```diff
@@ -1,22 +1,26 @@
 const POST_GRAPHQL_FIELDS = `
 sys {
+  firstPublishedAt
```

Comment:
just curious - what is the difference between firstPublishedAt and publishedAt. i thot it would be publishedAt and updatedAt?

Reply:
Good question. What we're using here are Contentful's naming conventions. Search for `publishedAt` on [this page](https://www.contentful.com/developers/docs/references/content-management-api/#/introduction/collection-resources-and-pagination) to see the explanations.